include/ofi_net.h

```diff
@@ -655,6 +655,7 @@ struct ofi_addr_list_entry {
 	char			ipstr[INET6_ADDRSTRLEN];
 	union ofi_sock_ip	ipaddr;
 	size_t			speed;
+	int			mtu;
```

Comment:
Please separate out adding / setting the mtu to ofi_addr_list_entry into its own patch.  That will make it clearer which changes are generic, versus tied specifically to udp.

Reply:
Makes sense. I'll do that with the next update.